docs/content/opa-1.md

```diff
@@ -0,0 +1,201 @@
+---
+title: OPA 1.0
+kind: misc
```

Comment:
I'd be inclined to file this under a more prominent section, even the core docs for the next few releases. We're going to promote this, but it'd be nice to have it be eye catching in the sidebar too.